addons/base_automation/static/tests/base_automation_error_dialog.js

```diff
@@ -0,0 +1,70 @@
+odoo.define('base_automation.BaseAutomatioErrorDialogTests', function (require) {
+"use strict";
+
+    const CrashManager = require('web.CrashManager').CrashManager;
+
+    QUnit.module('base_automation', {}, function () {
+
+        QUnit.module('Error Dialog');
+
+        QUnit.test('Error dialog with disable/edit automated actions buttons', async function (assert) {
+            assert.expect(4);
+
+            let baseAutomationName = "Test base automation error dialog";
+            let error = {
+                type: "Odoo Client Error",
+                message: "Message",
+                data: {
+                    debug: 'Traceback',
+                    context: {
+                        exception_class: 'base_automation',
+                        base_automation: {
+                            id: 1,
+                            name: baseAutomationName,
+                            is_admin: true,
+                        },
+                    },
+                },
+            };
+            let crashManager = new CrashManager();
+            let dialog = crashManager.show_error(error);
+
+            await dialog._opened;
+
+            let $el = dialog.$el;
+
+            assert.strictEqual($el.find('.o_clipboard_button').length, 1, "should display Copy full error button");
```

Comment:
you could check in the DOM directly, and use `containsOnce`, `containsN` and `containsNone`. For instance:
```
assert.containsOnce(document.body, '.modal');
assert.containsOnce(document.body, '.modal .o_clipboard_button');
assert.containsOnce(document.body, '.modal .o_disable_action_button');
assert.containsOnce(document.body, '.modal .o_edit_action_button');
```